lib/first_run.js

```diff
@@ -43,6 +43,15 @@ var firstRun = {
       require("sdk/tabs").open(
         "chrome://privly/content/privly-applications/Pages/ChromeFirstRun.html");
     }
+  },
+
+  /**
+   * Returns the coverage data for this file/module when tested.
+   */
+  coverage: function() {
+    var cv = require("./coverage_var.js").coverageVar;
+    return eval(cv.generate("first_run.js"));
```

Comment:
Could this be solved by using bracket notation? `this[cv.generate("first_run.js")]` or something similar?